infrastructure/networking.tf

```diff
@@ -49,6 +49,22 @@ resource "azurerm_virtual_network_peering" "tooling_to_bo" {
   provider = azurerm.tooling
 }
 
+resource "azurerm_virtual_network_peering" "bo_to_front_office" {
+  name                      = "${local.org}-peer-${local.service_name}-to-front_office-${var.environment}"
+  remote_virtual_network_id = data.azurerm_virtual_network.front_office_vnet.id
+  resource_group_name       = azurerm_virtual_network.main.resource_group_name
+  virtual_network_name      = azurerm_virtual_network.main.name
+}
+
+resource "azurerm_virtual_network_peering" "front_office_to_bo" {
+  name                      = "${local.org}-peer-tooling-to-${local.service_name}-${var.environment}"
```

Comment:
We should replace tooling here, else it'll be the same name as the tooling peering.

`tooling` -> `front-office` to be consistent with the other one?